test/test.js

```diff
@@ -162,13 +149,15 @@ try {
     if (sentry.length) logOnDetails.sha_sentryfile = sentry;
 }
 catch (beef){
-    util.log("Cannae load the sentry. " + beef);
+    util.log("Cannot load the sentry. " + beef);
```

Comment:
There is this great thing called `git blame`. It allows us to apportion blame for specific lines of code, like this one:

http://i.imgur.com/dXSEXCC.png

What a monster, indeed.